arrows/vxl/image_io.cxx

```diff
@@ -138,6 +143,111 @@ convert_image_helper(const vil_image_view<bool>& src,
   dest = src;
 }
 
+std::string
```

Comment:
I realize separator lines are intermittent throughout this file, but it would be nice to add them to new code.